pytorch_lightning/trainer/trainer.py

```diff
@@ -911,7 +913,37 @@ def call_hook(self, hook_name, *args, capture=False, **kwargs):
     @staticmethod
     def available_plugins():
         """
-            List of all available plugins that can be string arguments to the trainer.
-            Returns: List of all available plugins that are supported as string arguments.
+        List of all available plugins that can be string arguments to the trainer.
+        Returns: List of all available plugins that are supported as string arguments.
         """
         return PluginConnector.available_plugins()
+
+    @staticmethod
+    def get_gpu_memory_map(name=None):
```

Comment:
This doesn't seem necessary for this PR, and profiling/memory outputs needs to be discussed after

Reply:
agree ! Great catch ! Forgot to remove it.